### PR TITLE
Feature/ete test multiple backends

### PR DESCRIPTION
**Pull request by SerialVelocity**
Modify ETE tests so that they can be run backed by multiple backends

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/615)

### CODE REVIEW

File: atlasdb-ete-tests/build.gradle [hunk 1]
```diff
@@ -21,6 +21,10 @@ dependencies {
     compile "io.dropwizard:dropwizard-core:" + libVersions.dropwizard
     compile 'com.google.dagger:dagger:2.0.2'
 
+    runtime(project(':atlasdb-dbkvs')) {
+        exclude(module:'HikariCP')
```
Comment (jboreiko) [inline atlasdb-ete-tests/build.gradle]: What is this exclude for?  Are these not the same all using the same Hikari version?

Comment (SerialVelocity) [inline atlasdb-ete-tests/build.gradle]: Nope. atlasdb-hikari uses 2.4.3, everything else 2.4.5

**Comment by jboreiko** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/atlasdb/615)*

**Comment by SerialVelocity** [comment]
Review status: 12 of 14 files reviewed at latest revision, 1 unresolved discussion.

---

*[atlasdb-ete-tests/build.gradle, line 25 \[r1\]](https://reviewable.io:443/reviews/palantir/atlasdb/615#-KLcA-dZJkVTYBlVYt2B-r1-25:-KMUK2ihnZckyzIlyk_J:1409917535) ([raw file](https://github.com/palantir/atlasdb/blob/000b9b3cd61ee2dddf2179d7900ec2e04102723a/atlasdb-ete-tests/build.gradle#L25)):*
<details><summary><i>Previously, jboreiko (John Boreiko) wrote…</i></summary>
> > Modify ETE tests so that they can be run backed by multiple backends
> > 1 WEEK AGO
> > SerialVelocityBen Grabham
> I'm wondering if we should just bump our version, might want to have as a separate PR and see how the internal product reacts.  Let's see what @clockfort thinks
</details>
Different PR removed the version conflict. Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/atlasdb/615)*

**Comment by SerialVelocity** [comment]
Reviewed 1 of 2 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/atlasdb/615)*

**Comment by jboreiko** [comment]
:lgtm: +(pr: +1) 

---

Reviewed 1 of 2 files at r3, 1 of 1 files at r4.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/atlasdb/615#-:-KMUW0wEJYLvU_a4NUjL:1068775089)*

**Pull request merged**